string item_id
float32 height
float32 width
float32 breadth